src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -1602,85 +1602,121 @@ public with sharing class AFFL_MultiRecordType_TEST {
 
     }
 
+    /**
+     * @description Tests whether the record type labels are cached and matches the size of record types for Account
```

Comment:
Like the test for limits. As a complete side note, this test class is over 2k lines and might be already out of control. We should  put it on top of our refactoring list. (maybe we already did)

Reply:
This class is unusually large and its permutations are complicated. I feel like when we move to a better architecture, we will be able to test these cases separately with more confidence.